config-service-api/src/main/proto/org/hypertrace/config/service/v1/config_service.proto

```diff
@@ -119,6 +121,31 @@ message DeleteConfigResponse {
   ContextSpecificConfig deleted_config = 1;
 }
 
+message DeleteConfigsRequest {
+  repeated ConfigToDelete configs = 1;
+
+  message ConfigToDelete {
+    // required - name of the resource associated with the config
+    string resource_name = 1;
+    // required - namespace with which the config resource is associated
+    string resource_namespace = 2;
+    // optional - only required if config applies to a specific context.
+    // If empty, specified config is associated with a default context.
+    string context = 3;
+  }
+}
+
+message DeleteConfigsResponse {
+  repeated DeletedConfig deleted_configs = 1;
+
+  message DeletedConfig {
```

Comment:
The singular API returns a `ContextSpecificConfig` - why not reuse that?

Reply:
I was contemplating between using it itself and a separate config, but went the way of `UpsertAllConfigs` finally. Updated it now.